packages/manager/modules/enterprise-cloud-database/src/create/choose-database/choose-database.constants.js

```diff
@@ -0,0 +1,3 @@
+import PG_SQL_ICON from '../../assets/pgsql.png';
+
+export default PG_SQL_ICON;
```

Comment:
```suggestion
export const PG_SQL_ICON = PG_SQL_ICON;

export default {
PG_SQL_ICON;
}
```
you might want to add constants later on